src/main/java/net/sf/jabref/gui/desktop/os/Windows.java

```diff
@@ -48,8 +48,6 @@ public void openFolderAndSelectFile(String filePath) throws IOException {
 
     @Override
     public void openConsole(String absolutePath) throws IOException {
-        ProcessBuilder process = new ProcessBuilder("cmd.exe", "/c", "start");
-        process.directory(new File(absolutePath));
-        process.start();
+        Runtime.getRuntime().exec("cmd.exe /c start", null, new File(absolutePath));
```

Comment:
Be careful with that! 
It was change to PB with intention in https://github.com/tschechlovdev/jabref/commit/73f69e6c2a6ddcaef8a14207a1967f813c3fdf24